spec/ReviewsSummary.test.js

```diff
@@ -5,10 +5,13 @@ import 'jest-enzyme';
 import 'jest-styled-components';
 import { mount } from 'enzyme';
 
-import { reviews } from '../__mocks__/mockData.js';
-import ReviewsSummary from '../client/src/components/ReviewsSummary.jsx';
+import { reviews } from '../__mocks__/mockData.js'; //eslint-disable-line
```

Comment:
yeah I think eslint doesn't like the '.js'. If you want to fix it instead of adding all these comments around, add this object to the top level of your webpack config:
`  
resolve: {
    extensions: ['.jsx', '.js'],
  }
`
(check mine if that doesn't make sense)

I don't really care either way though, that's just if you're annoyed by all the eslint disables